doc/rst/language/spec/statements.rst

```diff
@@ -856,9 +857,12 @@ enumerated type. It is an error to provide a name in a
 ``limitation-clause`` that does not exist or is not visible in the
 respective module or enumerated type.
 
-If a type is specified in the ``limitation-clause``, then the type’s
-fields and methods are treated similarly to the type name. These fields
-and methods cannot be specified in a ``limitation-clause`` on their own.
+If a type or type's secondary methods are defined in the used module, then any
+instances of the type obtained in the scope of the use may access the fields and
+methods of that type, regardless of the ``limitation-clause``. These fields
+and methods cannot be specified in a ``limitation-clause`` on their own.  The
+privacy of use statements is also ignored when determining if an instance can
+access the fields and methods, for similar reasons.
```

Comment:
Just wanted to say that this paragraph is infinitely better than the one we currently have

Reply:
We've also changed the behavior this release, forgot to mention that in my PR description (updating now)